user/views.py

```diff
@@ -1,6 +1,5 @@
 from rest_framework import generics
 from rest_framework.authentication import TokenAuthentication
-from rest_framework.permissions import IsAuthenticated
 from rest_framework.authtoken.views import ObtainAuthToken
```

Comment:
Remove this import and delete CreateTokenView as we use JWT in this task + remove AuthTokenSerializer as it's used in this view only